### ISSUE TITLE
demo

### ISSUE THREAD

**Issue by xiaotianzhen** [ShengCN/PixHtLab-Src#18]
Your demo is inconsistent with the interface shown in the demonstration instructions. How can I use the demo?

**Comment by ShengCN** [comment]
Are you referring to PixhtLab? What do you mean by "inconsistent"?

**Comment by xiaotianzhen** [comment]
Yes, the inconsistency I am talking about is that the graphical interface displayed in the readme is inconsistent with the display of the demo I ran.

**Comment by LukaGiorgadze** [comment]
@xiaotianzhen How did you run the demo? I was following the steps described in README but getting this error https://github.com/ShengCN/PixHtLab-Src/issues/20

**Comment by xiaotianzhen** [comment]
@LukaGiorgadze I just ran the demo and the problem you mentioned did not occur. Let’s see if there is a problem with the path.

**Comment by LukaGiorgadze** [comment]
@ShengCN That's the [result](https://github.com/ShengCN/PixHtLab-Src/assets/6573695/68e2bd4a-d2e7-47dd-a9c4-72cc076b886a) I received from the demo code, is it expected? Or shadow should be softer and smoother?

Second question - how can I generate depth of image? is it possible to generate shadows without depth maps as in SSN?

**Comment by foocker** [comment]
how to get the pure shadow as a template to blending to the other image? only hwo to get the pure shadow image.
